app/views/api/integrations/apicast/configuration_driven/_configuration.html.slim

```diff
@@ -22,9 +22,39 @@
 
     div class=("feedback api #{'success' if @proxy.api_test_success} #{'no-test' if @proxy.api_test_path.blank? or @proxy.api_test_success.nil?}")
       i.fa.fa-puzzle-piece
-      = f.inputs "API" do
-        = f.input :api_backend, input_html: { data: { default: @proxy.default_api_backend }, autofocus: (@proxy.api_backend == @proxy.default_api_backend) }, hint: is_https?(@proxy.api_backend) ? t('formtastic.hints.proxy.api_backend_https') : t('formtastic.hints.proxy.api_backend_http')
-        = help_bubble( 'api_endpoint_bubble', t('api.integrations.proxy.api_endpoint_help_html'))
+      - if @service.acts_as_product?
+        = f.inputs 'Backends' do
+          table#backend_apis
+            thead
+              tr
+                th Name
+                th Production Public Base URL
+                th
+                th Private Base URL
+                th
+            tbody
+              = f.fields_for :backend_api_configs do |backend|
+                - backend_api = backend.object.backend_api
+                tr
+                  td
+                    = link_to provider_admin_backend_api_path(backend_api) do
+                      => backend_api.name
+                  td
+                    = @proxy.default_production_endpoint
+                    | /
+                    = backend.text_field :path, class: 'backend_api_config_path'
+                  td
+                    i.fa.fa-caret-right
+                  td
+                    = backend_api.private_endpoint
+                  td.actions
+                    = link_to '', '#delete-backend-api-config', class: 'action delete'
+                    = backend.hidden_field :_destroy, class: 'destroyer', value: 1, disabled: true
+
+      - else
+        = f.inputs "API" do
+          = f.input :api_backend, input_html: { data: { default: @proxy.default_api_backend }, autofocus: (@proxy.api_backend == @proxy.default_api_backend) }, hint: is_https?(@proxy.api_backend) ? t('formtastic.hints.proxy.api_backend_https') : t('formtastic.hints.proxy.api_backend_http')
```

Comment:
I see that `app/views/api/integrations/apicast/script_driven/_staging.html.slim` also uses this:

```
hint: is_https?(@proxy.api_backend) ? t('formtastic.hints.proxy.api_backend_https') : t('formtastic.hints.proxy.api_backend_http')
```

WDYT in extracting this into a helper?

```
.. hint: uri_hint(@proxy.api_backend) ...
```

and in the helper:
```
def uri_hint(uri)
   is_https?(uri) ? t('formtastic.hints.proxy.api_backend_https') : t('formtastic.hints.proxy.api_backend_http')
end
```